pkg/controller/sync/status/status.go

```diff
@@ -139,6 +147,8 @@ func SetFederatedStatus(fedObject *unstructured.Unstructured, reason AggregateRe
 	if err != nil {
 		return false, errors.Wrapf(err, "Failed to marshall generic resource json to unstructured")
 	}
+
+	klog.Infof("Setting the status of federated object '%v' and resource object '%v'", fedObject.GetName(), resourceObj.GetName())
```

Comment:
I still see many `per resource` messages at loglevel 0. As mentioned in [this comment](https://github.com/kubernetes-sigs/kubefed/pull/1292#discussion_r504540891) it does not make sense to have per resource messages at loglevel zero. This has the potential  of flooding the logs for no reason. There would be no way for the user to reduce the logs generated if debug messages as above are set at this level. These should be level 4 or above.

Only once in a lifecycle message, for example controller being initiated/shutdown can be at this level.

Please update across this PR.

Reply:
If there are some errors in collecting status or setting that, that you want  reported, I think that can be reported as level  2 info. The actual error should eventually surface in the status conditions of the resource.

Reply:
@irfanurrehman I changed the log verbosity accordingly. Mainly, I set all to V(4).